src/components/sections/Catalogue/CurrentProductSection/components/Articles/ProductDescription/components/RadioBtnGroup.tsx

```
@@ -0,0 +1,51 @@
+            className='flex items-center'>
+            <RadioButton
+              classNameLabel={`shrink-0 flex relative size-5 rounded-full items-center justify-center border-2
+            text-sm ${selectedSample === title ? 'border-brown-light ' : 'border-grey-normal'} hover:bg-brown-accent/50 `}
+              value={title}
+              selected={selectedSample}
+              name='sample'
+              onChange={handleVariantChange}
+              classNameInput='absolute inset-0 cursor-pointer opacity-0'>
+              <div
+                className={`absolute z-10 size-3 shrink-0 rounded-full hover:size-8 hover:bg-brown-accent/20
+                ${selectedSample === title ? 'bg-brown-light' : 'bg-white'}`}></div>
```

ESLint

[error] Classname 'hover:bg-brown-accent/20' is not a Tailwind CSS class!

(tailwindcss/no-custom-classname)